src/include/sof/task.h

```diff
@@ -36,7 +36,9 @@
 struct sof;
 struct task;
 
-int do_task(struct sof *sof);
+int do_task_master_core(struct sof *sof);
```

Comment:
probably better if we can pass the core ID in here an use a single API call. We can also define PLATFORM_MASTER_CORE 0 if needed too